cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -80,6 +79,21 @@ object CassandraLauncher {
   }
 
   /**
+   * Use this to add classpath directories to the classpath
```

Comment:
I guess this is used if custom logging or configResource yaml is neeeded. Please add that to this as examples.